builder/docker/assemblyscript/Dockerfile

```diff
@@ -1,9 +1,6 @@
 FROM suborbital/subo:dev as subo
 
 FROM node:16-buster-slim
-
-RUN npm install -g npm@latest
-
+WORKDIR /root/runnable
 COPY --from=subo /go/bin/subo /usr/local/bin
-
-WORKDIR /root/runnable
\ No newline at end of file
+RUN npm install -g npm@latest
```

Comment:
Just for my own curiosity, was moving this line below the `COPY` significant for some weird Docker caching/optimization reason?

Reply:
In this case it shouldn't really matter, but ya it's always best to have runs at the bottom to maintain prevent cache busting.